apps/src/lab2/views/components/Instructions/ResourcePanel/styles.module.scss

```diff
@@ -18,21 +18,15 @@
     flex-direction: column;
   }
 
-  .tabButton {
-    @include remove-button-styles;
```

Comment:
Can we remove the `mixins` import as well?

Reply:
good call! We can actually remove the color import too; that import is always available now.